packages/experimental-nextjs-app-support/README.md

```diff
@@ -154,6 +158,60 @@ export default function RootLayout({
 
 If you want to make the most of the streaming SSR features offered by React & the Next.js App Router, consider using the [`useSuspenseQuery`](https://www.apollographql.com/docs/react/api/react/hooks-experimental/#using-usesuspensequery_experimental) and [`useFragment`](https://www.apollographql.com/docs/react/api/react/hooks-experimental/#using-usefragment_experimental) hooks.
 
+### Preloading data in RSC for usage in Client Components
+
+You can also preload data in RSC to populate the cache of your Client Components.
+
+For that, follow the setup steps for both RSC and Client Components as laid out in the last two paragraphs.
+
+Then you can use the `PreloadQuery` component in your React Server Components:
```

Comment:
```suggestion
For that, follow the setup steps for both RSC and Client Components as laid out in the last two paragraphs.

Then you can use the `PreloadQuery` component in your Server Components:
```

The [React docs](https://react.dev/reference/rsc/server-components) tend to use "Server Components" without the word "React" when referring to them so perhaps we should do the same?

Reply:
I prefer to be clear here, as we also use "RSC" in places.